utils/elrond-go-keygenerator

```diff
@@ -10,7 +10,8 @@ WORKDIR /go/elrond-go/cmd/keygenerator
 RUN go build
 
 # ===== SECOND STAGE ======
-FROM ubuntu:18.04
+FROM ubuntu:22.04
```

Comment:
is there a limitation to use go 1.14.9?
I would change to 1.17.6 as well

Reply:
why not bump it to go 1.19.3 which is the latest version of golang image?

Reply:
used 1.17.6, since it's currently the go version we support everywhere